### PR TITLE
feat: add new spacing variables

### PR DESCRIPTION
**Pull request by felipefialho**
## Infos

[Task](https://juntossomosmais.monday.com/boards/2413187753/pulses/2421279082)

#### What is being delivered?

A new CSS variables of spacings to be used inside projects

#### What impacts?

This task should not cause collateral damage because the old variables were kept and changes to projects need to be made gradually

#### Reversal plan

N/A

#### Evidences

image

### CODE REVIEW

**Review by MayaraRMA — APPROVED** [review]

**Review by deverebor — DISMISSED** [review]

File: packages/styles/global/variables.scss [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+  --spacing-large: 24px;
+  --spacing-xlarge: 29px;
+  --spacing-xxlarge: 32px;
+  --spacing-xxxlarge: 40px;
+  --spacing-xxxxlarge: 48px;
+  --spacing-giant: 64px;
+  --spacing-xgiant: 80px;
 
+  /* Colors
+  * ------------------------- */
```
[the remark this answers is not in the document]
Comment (felipefialho) [reply packages/styles/global/variables.scss:18]: No! I just add a comment to separate the blocks

File: packages/styles/global/variables.scss [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-  --screen-max-container: 1140px;
+  /* Spacings
+  * ------------------------- */
+  --spacing-xxsmall: 4px;
+  --spacing-xsmall: 8px;
+  --spacing-small: 12px;
+  --spacing-base: 16px;
+  --spacing-medium: 20px;
+  --spacing-large: 24px;
+  --spacing-xlarge: 29px;
```
Comment (mauriciomutte) [inline packages/styles/global/variables.scss:10]: question: this is correct? shouldn't it be 28px?

**Review by munyrasirio — DISMISSED** [review]
Nice! ❤️

File: packages/styles/global/variables.scss (hunk 2 above)
Comment (felipefialho) [reply packages/styles/global/variables.scss:10]: THANKS! hahahaha 😱

**Review by mauriciomutte — DISMISSED** [review]
Great! 🚀

**Review by munyrasirio — APPROVED** [review]

**Review by tassioFront — APPROVED** [review]

**Review by mauriciomutte — APPROVED** [review]

File: packages/styles/global/variables.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (deverebor) [reply packages/styles/global/variables.scss:18]: Ok!

File: packages/styles/global/variables.scss [hunk 3]
```diff
@@ -131,4 +148,15 @@
   --typography-caption-letter: -0.171429px;
   --typography-button-font: 500 16px/19px var(--font-family);
   --typography-button-letter: -0.228572px;
+
+  /* OLD: Spacings
+   * TODO: Remove it after exclude from projects
+  * ------------------------- */
+  --space-xxs: 4px;
```
[the remark this answers is not in the document]
Comment (felipefialho) [reply packages/styles/global/variables.scss:155]: Yes! Maybe is a good idea 😁

**Pull request merged**